Scripts/Modules/TooltipModule/TooltipModule.cs

```diff
@@ -177,14 +188,27 @@ void UpdateVisuals(ITooltip tooltip, TooltipUI tooltipUI, Transform target, floa
 
 		public void OnRayEntered(GameObject gameObject, RayEventData eventData)
 		{
+			if (gameObject == this.gameObject)
+				return;
+
+			var tooltip = gameObject.GetComponent<ITooltip>();
+			if (tooltip != null)
+				ShowTooltip(tooltip);
+		}
+
+		public void OnRayHovering(GameObject gameObject, RayEventData eventData)
+		{
+			if (gameObject == this.gameObject)
+				return;
+
 			var tooltip = gameObject.GetComponent<ITooltip>();
 			if (tooltip != null)
```

Comment:
I just found another bug related to this tooltip which I couldn't find an unobtrusive solution for, and therefore probably will be introduced by this PR:

If you have a tool open, hovering the pinned tool button, which is a child of the main menu activator, you see both tooltips. I'd like a code solution, i.e. not letting tooltip activations "bubble up" past the first valid new tooltip, but this would involve major changes to the input module.

Also, this is all going to change in the toolflow PR.